samples/MobileBuyIntegration/app/src/main/java/com/shopify/checkout_sdk_mobile_buy_integration_sample/logs/LogOverview.kt

```diff
@@ -33,27 +31,23 @@ import androidx.compose.material.TextButton
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
-import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.text.style.TextDecoration
-import com.shopify.checkout_sdk_mobile_buy_integration_sample.logs.Logs.DATE_FORMAT
 import com.shopify.checkout_sdk_mobile_buy_integration_sample.logs.Logs.OVERVIEW_FONT_SIZE
-import java.util.Date
 
 @Composable
-fun LogOverviewRow(logDate: LogDate, logMessage: LogMessage, color: Color, onClick: () -> Unit) {
```

Comment:
There are exceptions, but in Compose, passing a ``Modifier` is often a better alternative than passing specific properties such as `color`.